src/tir/transforms/unroll_loop.cc

```diff
@@ -57,6 +58,9 @@ struct UnrollLoopConfigNode : public tvm::AttrsNode<UnrollLoopConfigNode> {
     TVM_ATTR_FIELD(explicit_unroll)
         .describe("Whether to explicitly unroll the loop instead of setting a pragma")
         .set_default(true);
+    TVM_ATTR_FIELD(explicit_unroll_max_extent)
+        .describe("The maximum extent of a loop that can be unrolled explicitly (-1 for infinite)")
+        .set_default(32);
```

Comment:
How about change the default value to `16`? As like ARM CPU (especially arm 32 bits), 32 is still a little bigger for them. `16` should cover common hardwares.

Reply:
What's the performance impact of 16 on X86? Should we check the related TOPI templates, including X86 and ARM, for the performance impact?

Reply:
I think 32 is a very safe value. How do you know 32 is too large for ARM CPU?
This value is to avoid the llvm hanging issue, not for performance.
The default value should be large enough to avoid performance regression.

Reply:
Thanks for the explanation and I agree with your point. I just realized that this is a global config instead of the `unroll_explicit` pragma in the CUDA templates.